docs/user-docs/test_matrix.md

```diff
@@ -34,23 +36,13 @@ Devices and configurations on which TrenchBoot is known to not work:
 |:---------------------------------------------------------------------------------------------:|:----------:|:----------:|:-----:|
 | Supermicro M11SDV-8CT<br>(AMD EPYC 3000 Snowy Owl server)<br>(v0.4 by openQA)                 |  TPM 2.0   | 2019-today | CSM legacy boot,<br>unexpected PCR values |
 
-[aem]: https://github.com/TrenchBoot/qubes-antievilmaid
```

Comment:
I think that keeping those close to where they are referenced make it easier to work with (modify, add new entries, move a part of document elsewhere etc.). Would it be possible to disable whatever check suggested this change?

Reply:
From what I can see currently nothing complains about that, so I'm not sure if @EduKav1813 wasn't overeager or maybe it was victim of another auto fix.

Changed it back here: 9611f9774549a0cd30d4c2338c5017f8dfef3c32